package.json

```diff
@@ -8,9 +8,11 @@
     "node": ">=4.0.0"
   },
   "repository": "googleapis/nodejs-common",
-  "main": "./src/index.js",
+  "main": "./build/src/index.js",
+  "types": "./build/src/index.d.ts",
   "files": [
-    "src",
+    "build/src",
+    "package.json",
```

Comment:
`package.json` is automatically included, but explicitly listing it looks good to me.

The [document](https://docs.npmjs.com/files/package.json#files) on npmjs.com says `README` is also always included but I'm not sure if it applies to `README.md`. Can you check or list it here too?

Reply:
It does. READMEs, LICENSEs, and package.json are auto-included 👍